tests/e2e/features/createSpace.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Create a space
+    As a super user
+    I want to create a space
+    So that I can add recipes and share my recipes
+
+
+    Background:
+        Given the user has signed up with the following details
+            | username         | superuser     |
```

Comment:
Is there any possibility to sign up a user via API? If there is then sign-up should be done using API, which is better and faster than via webUI